### PR TITLE
Added braincert_server_region setting to virtual classrooms

### PR DESCRIPTION
**Pull request by xyc0562**
Added a braincert_server_region option to virtual classrooms component settings, with default set to Singapore.

### CODE REVIEW

File: config/locales/en/course/admin/virtual_classroom_settings.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ en:
           title_placeholder: 'Leave blank to use default title'
           braincert_whiteboard_api_key_hint: >
             Braincert Virtual Classroom API Key for the Virtual Classroom Component
+          braincert_server_region_hint: >
+            Braincert Virtual Classroom Server Region
```
Comment (weiqingtoh) [inline config/locales/en/course/admin/virtual_classroom_settings.yml]: You might want to add that this is probably an integer value?

File: app/models/course/virtual_classroom_settings.rb [hunk 2]
```diff
@@ -27,6 +27,20 @@ def braincert_whiteboard_api_key=(value)
     @settings.braincert_whiteboard_api_key = value
   end
 
+  # Returns BrainCert Virtual Classroom server region
+  #
+  # @return [String] The custom or default title of virtual classrooms component
+  def braincert_server_region
+    @settings.braincert_server_region || 7 # 7 is code for Singapore
```
Comment (weiqingtoh) [inline app/models/course/virtual_classroom_settings.rb]: I think `@settings.braincert_server_region` is a string, does it matter if it's a string or a integer?

Comment (xyc0562) [inline app/models/course/virtual_classroom_settings.rb]: Hi Weiqing,

For now it's an integer. I am not sure whether that may be changed by BrainCert side in the future though.

What would your suggestion be? Should I use integer input or just normal text input?

Comment (weiqingtoh) [inline app/models/course/virtual_classroom_settings.rb]: I'm ok either way - main focus is to ensure maintainability.

What you can also do is the do the default value setting at the service, so if the braincert API changes,  only the service needs to be changed.

Comment (xyc0562) [inline app/models/course/virtual_classroom_settings.rb]: I think in this case we can just leave it as is. Not very fond of moving default to service as it seems more coherent to store settings defaults in a single place.

Comment (weiqingtoh) [inline app/models/course/virtual_classroom_settings.rb]: 👍

**Comment by weiqingtoh** [comment]
@xyc0562 can rebase and push?

**Pull request merged**